Makefile

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+SHELL=/bin/bash -o pipefail
+
+PROJECT_NAME=zookeeper-operator
+REPO=pravega/$(PROJECT_NAME)
+VERSION=$(shell git describe --always --tags --dirty)
+GIT_SHA=$(shell git rev-parse --short HEAD)
```

Comment:
Is this required?  The `VERSION` from `git describe` has the HEAD commit SHA at the end

Reply:
Actually, `VERSION` will contain the commit SHA if there's no tag available. Otherwise it'll refer to the closest tag. After the first release, there will always be a tag to be referenced. 

However, I now realized that the commit SHA is also added when new commits are added on top of the tag. In the last command, I would like to see `0.0.0-1` instead of `0.0.0-1-g8c44b62`. 

@maddisondavid any ideas?

```
$ git describe --always --tags --dirty
876567a
$ git tag 0.0.0
$ git describe --always --tags --dirty
0.0.0
$ touch foo
$ git add foo
$ git describe --always --tags --dirty
0.0.0-dirty
$ git commit -m "Add foo"
$ git describe --always --tags --dirty
0.0.0-1-g8c44b62
```

Reply:
I've fixed the above issue in the last commit.